circuits/cpp/barretenberg/cpp/src/barretenberg/proof_system/arithmetization/gate_data.hpp

```diff
@@ -127,4 +127,11 @@ template <typename FF> struct ecc_add_gate_ {
     FF endomorphism_coefficient;
     FF sign_coefficient;
 };
+
```

Comment:
(x1, y1) is the input point and (x3, y3) is the output point presumably -- if so, can you add a doc comment stating this?

Reply:
nit: Oh I see its commented on top of `create_ecc_dbl_gate` so its probably better to avoid documentation duplication, you could probably do:

```
template <typename FF> struct ecc_dbl_gate_ {
     uint32_t input_x;
     uint32_t input_y;
     uint32_t output_x;
     uint32_t output_y;
 };
 ```